core/templates/dev/head/css/oppia.css

```diff
@@ -2788,6 +2788,59 @@ div#ng-curtain {
   margin-left: -3px;
 }
 
+/* Oppia Exploration Switch Button */
+.switch-button-down {
```

Comment:
Any classes we introduce need to be prefixed with .oppia- to distinguish them from third-party classes.